### DISCUSSION TITLE
关于浏览器缩略图的一个小BUG，请求修复！

### DISCUSSION THREAD

**Discussion by Design-W** [AlUlkesh/stable-diffusion-webui-images-browser#171] [General]
关于浏览器缩略图的一个小BUG，请求修复！
[点开点开任何一个浏览项，当缩略图的数量超过视图框的范围，最下排的小缩略图无法将本页内所有的图片显示出来，而且无法左右移动至首尾处，无法通过最下排的小缩略图去选择浏览的图片，此BUG是更新了最新的开发版WEB UI后出现的问题！commit: 5ab7f213

![QQ截图20230510160632](https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/assets/129763511/fe94e0ce-3691-495d-b429-804519993bbc)
![QQ截图20230510160655](https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/assets/129763511/cefc0711-fb0d-4d9a-a272-7a8dba255994)

**Comment by AlUlkesh** [comment]
Apparently one of the Gradio updates changed something. Should work again, if you update: https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/commit/5f9e5ad49035b6237f6c20f528d9313bb41e6f75

**Comment by Design-W** [comment]
还有一个小BUG：我点右侧的小箭头展开Additional Networks选项后，然后选择需求的lora，我再点击箭头，收起扩展。当再一次需展开时，界面会出BUG，只会显示一个lora选项，而且此时界面内所有选项都无法操作，必须刷新WEB UI，才可以继续使用，似乎和收起左边的模型选项卡有关？
![QQ截图20230515014902](https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/assets/129763511/c48d398d-037b-45ba-9d83-c01a4cb6834f)
![QQ截图20230515015233](https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/assets/129763511/e1b329e9-c94e-4e42-bdad-4cb67750e16c)
